roles/online_docs/templates/mkdocs/docs/logins-windows.md

```diff
@@ -39,55 +47,121 @@ If you prefer another terminal application consult the corresponding manual.
+ - other web browsers: the download itself is easier, but when executing you get `Windows protected` warning, and you must click on `More info` text (small text at top left part of the window), then select `Run anyway`.
+ - alternatively, you can simply click on [the link](../logins-windows.bat), then select all the text, copy and paste into a filename named `logins-windows.bat` (the `.bat` ending is mandatory in order the file to become executable)
+
+Once you have the file, run it and the rest of the configuration will be (for except two prompts) done almost automatically.
+
+The configuration files will be stored inside the `C:\Users\[Username]\.ssh\` folder.
+
+### 2.1 Manual configuration with
+
+(to be updated)
+
+The ssh configuration files can be created inside the `C:\Users\[Username]\.ssh\` folder. In order to manually create them, you can simply follow the instructions for [linux ssh configuration](../logins-linux-config/).
+
+There are **many** differences between the linux and windows OpenSSH configuration. One of the biggest is the difference beween linux `/` and windows `\` as directory path separator. But there are many more. Users that do not understand those differences, should use the MobaXterm or semi-automatic .bat script options above.
+
+
+### 2.3 Connecting to the system
+
+In order to connect to the {{ slurm_cluster_name }}
+
+- first open Start menu, search and execute the `cmd` or `Command Prompt` program
+- login to {{ slurm_cluster_name }} by using a command `ssh umcg-username@{{ groups['jumphost'] | first }}+{{ groups['user_interface'] | first }}`
+
+### 2.4 Port forwarding to the compute nodes
+
+Users can connect ports on own computer to the ports on the compute nodes behind the jumphosts - by using port forwarding (binding).
+In order to do this, user must open another `cmd` environment and establish a connection with numbers of the ports to be forwarded
+
+    ssh -J {{ groups['jumphost'] | first }} {{ groups['user_interface'] | first }} -L XXXXX:NODE:XXXXX
+
+where
+
+- `-J {{ groups['jumphost'] | first }}` specify which jumphost machine is going to be used in order oppening the connection to ...
+- `{{ groups['user_interface'] | first }}` is the name of the machine that user will connect and have the command line started
+- `-L XXXXX:NODE:XXXXX` defines the ports and machines to connect
+    - `XXXXX` can be set to any port in the range (1024 < `XXXXX` <= 65535) as long it is **free** on both (your local and remote) servers,
+    - note that this example binds the **same** `XXXXX` ports on both local as well as remote server - but this is not mandatory, they can be different,
+    - `NODE` is the name of the remote server behind a jumphsot to which the ports should be forwarded.
+
+More information about `ssh` options can be find on the [ssh man pages](https://man.openbsd.org/ssh).
+
+### 2.5 Converting Putty .ppk key into OpenSSH format
```

Comment:
If OpenSSH on Windows works well, then I suggest to use OpenSSH also for generating key pairs and drop all MobaXterm references from the documentation.

Reply:
added